src/host/VtInputThread.cpp

```diff
@@ -45,6 +46,27 @@ VtInputThread::VtInputThread(_In_ wil::unique_hfile hPipe,
 
     _pInputStateMachine = std::make_unique<StateMachine>(engine.release());
     THROW_IF_NULL_ALLOC(_pInputStateMachine.get());
+
+    _shutdownWatchdog = std::async(std::launch::async, [&] {

```

Comment:
It wasn't really easy to re-use because each watchdog had to do something slightly different. I'm just going to leave it given that I exhausted my stamina on this particular change about 21 days ago and I don't want to fiddle with it now that the tests prove it works.